docs/quickstart.md

```diff
@@ -254,8 +254,7 @@ if ($user->inGroup('admin', 'beta')) {
 
 ## Managing Users
 
-Shield uses a more complex user setup than many other systems, separating [User Identities](concepts.md#identities) from the user accounts themselves. This quick overview should help you feel more confident when working with users on a day-to-day basis.
-Since Shield uses a more complex user setup than many other systems, due to the [User Identities](concepts.md#user-identities), this quick overview should help you feel more confident when working with users on a day-to-day basis.
+Since Shield uses a more complex user setup than many other systems, separating [User Identities](concepts.md#identities) from the user accounts themselves, this quick overview should help you feel more confident when working with users on a day-to-day basis.
```

Comment:
```suggestion
Since Shield uses a more complex user setup than many other systems, separating [User Identities](concepts.md#user-identities) from the user accounts themselves, this quick overview should help you feel more confident when working with users on a day-to-day basis.
```